src/gc/gcconfig.h

```diff
@@ -95,6 +95,8 @@ class GCConfigStringHolder
       "Specifies the ratio compacting GCs vs sweeping")                                        \
   INT_CONFIG(GCHeapAffinitizeMask, "GCHeapAffinitizeMask", 0,                                  \
       "Specifies processor mask for Server GC threads")                                        \
+  STRING_CONFIG(GCHeapAffinitizeRanges, "GCHeapAffinitizeRanges",                              \
```

Comment:
>GCHeapAffinitizeRanges [](start = 16, length = 22)

I just noticed something I didn't notice before - somehow the string configs are treated differently from bool/int ones. usually when we add a config here we also add it in src\inc\clrconfigvalues.h but since string configs are just obtained on demand (instead of the VM side having to be aware of it) this works. this seems inconsistent to me. can we add this to clrconfigvalues.h just for consistency so if someone looks at that file they will also get all the configs?

Reply:
Good point